dali/ft/daftformat.cpp

```diff
@@ -1577,6 +1577,64 @@ offset_t XmlSplitter::getFooterLength(BufferedDirectReader & reader, offset_t si
 }
 
 
+CJsonInputPartitioner::CJsonInputPartitioner(const FileFormat & _format)
+{
+    format.set(_format);
```

Comment:
In HPCC-13045 I added LOG() for every spliter/partitoner constructor (to easy follow whats going on in spray). I think it would be good to do same in this new partitioner.